CryptoAnalysis/src/main/java/crypto/reporting/CommandLineReporter.java

```diff
@@ -1,112 +1,119 @@
-					if(marker instanceof ErrorWithObjectAllocation) {
-						 s += String.format(" (on Object #%s)\n", ((ErrorWithObjectAllocation) marker).getObjectLocation().getObjectId());
-					} else {
-						 s += "\n";
-					}
-					s += String.format("\t\t\t%s\n", marker.toErrorMarkerString());
-					s += String.format("\t\t\tat statement: %s\n\n", marker.getErrorLocation().getUnit().get());
-				}
-			}
-			s += "\n";
-		}
-		s += "======================= CogniCrypt Summary ==========================\n";
-		s += String.format("\tNumber of CrySL rules: %s\n", rules.size());
-		s += String.format("\tNumber of Objects Analyzed: %s\n", this.objects.size());
-		if(this.errorMarkers.rowKeySet().isEmpty()){
-			s += "No violation of any of the rules found.";
-		} else{
-			s += "\n\tCogniCrypt found the following violations. For details see description above.\n";
-			for(Entry<Class, Integer> e : errorMarkerCount.entrySet()){
-				s += String.format("\t%s: %s\n", e.getKey().getSimpleName(),e.getValue());
-			}
-		}
-		s += "=====================================================================";
-
-		System.out.println(s);
-		if (outputFolder != null) {
-			try {
-				FileWriter writer = new FileWriter(outputFolder +"/CogniCrypt-Report.txt");
-				writer.write(s);
-				writer.close();
-				for (SootClass c : this.errorMarkers.rowKeySet()) {
-					FileOutputStream streamOut = new FileOutputStream(new File(outputFolder +"/"+c.toString()+".jimple"));
-					PrintWriter writerOut = new PrintWriter(new EscapedWriter(new OutputStreamWriter(streamOut)));
-					Printer.v().printTo(c, writerOut);
-					writerOut.flush();
-					streamOut.close();
-					writerOut.close();
-				}
-			} catch (IOException e) {
-				throw new RuntimeException("Could not write to file " + outputFolder);
-			}
-		}
-	}
-}
+package crypto.reporting;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map.Entry;
+import java.util.Set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import crypto.HeadlessCryptoScanner.Format;
+import crypto.analysis.IAnalysisSeed;
+import crypto.analysis.errors.AbstractError;
+import crypto.analysis.errors.ErrorWithObjectAllocation;
+import crypto.rules.CrySLRule;
+import soot.Printer;
+import soot.SootClass;
+import soot.SootMethod;
+import soot.util.EscapedWriter;
+
+public class CommandLineReporter extends ErrorMarkerListener {
+	private static final Logger LOG = LoggerFactory.getLogger(CommandLineReporter.class);
+	private Format reportformat;
+	private File outputFolder;
+	private List<CrySLRule> rules;
+	private Collection<IAnalysisSeed> objects = new HashSet<>();
+
+	public CommandLineReporter(String string, Format format, List<CrySLRule> rules) {

```

Comment:
Rename first argument in this constructor

Reply:
Renamed the first argument of the constructor in CommandLineReporter and SARIFReporter